hal/src/b5som/network/quectel_ncp_client.cpp

```diff
@@ -114,7 +115,15 @@ int QuectelNcpClient::init(const NcpClientConfig& conf) {
     conf_ = static_cast<const CellularNcpClientConfig&>(conf);
 
     // Initialize serial stream
-    auto sconf = SERIAL_8N1 | SERIAL_FLOW_CONTROL_NONE; // TODO: replace with SERIAL_FLOW_CONTROL_RTS_CTS when HW is fixed.
+    auto sconf = SERIAL_8N1;
+
+    // Hardware version
+    // V003 - 0x00 (disable hwfc)
+    // V004 - 0x01 (enable hwfc)
+    if (hal_get_device_hw_version() > 0x00) {
```

Comment:
Let's do the opposite: if the revision is specifically V003 - disable hwfc, in all the other cases - enable.

Reply:
Fixed